RIAssigner/data/Data.py

```diff
@@ -29,38 +29,65 @@ def is_valid(value: Union[RetentionTimeType, RetentionIndexType]) -> bool:
         return result
 
     @staticmethod
-    def can_be_float(rt):
+    def can_be_float(rt: Union[Quantity, float, int]) -> bool:
+        """Determine whether a value can be converted to a float.
+
+        This function checks if the provided input is an instance of either
+        Quantity, float, or int.
+
+        Args:
+            rt (Union[Quantity, float, int]): Value to check for float conversion.
+
+        Returns:
+            bool: True if the input is an instance of Quantity, float, or int, False otherwise.
+        """
         if isinstance(rt, (Quantity, float, int)):
             return True
         return False
 
     @classmethod
-    def add_possible_rt_keys(cls, keys: List[str]):
-        """ A method that adds new identifiers for the retention time information lookup. """
-        cls._rt_possible_keys.update(keys)
+    def add_possible_rt_keys(cls, keys: List[str]) -> None:
+        """ A method that adds new identifiers to get retention time information.
+
+        Args:
+            keys (List[str]): A list of new identifiers (keys) to be added to the `_rt_possible_keys`.
+
+        Returns:
+            None
+        """
+        cls._rt_possible_keys.append(keys)
 
     @classmethod
-    def add_possible_ri_keys(cls, keys: List[str]):
-        """ A method that adds new identifiers for the retention index information lookup. """
-        cls._ri_possible_keys.update(keys)
+    def add_possible_ri_keys(cls, keys: List[str]) -> None:
+        """ A method that adds new identifiers to get retention index information. 
 
+        Args:
+            keys (List[str]): A list of new identifiers (keys) to be added to the `_ri_possible_keys`.
+
+        Returns:
+            None
+        """
+        cls._ri_possible_keys.append(keys)
+    
     @classmethod
     def get_possible_rt_keys(cls) -> List[str]:
-        """Method to get the supported retention time keys
-
+        """ A method that returns the possible keys to get retention times.
+        
         Returns:
-            List[str]: List of supported retention time keys.
+            List[str]:  A list of possible keys to get retention times.
         """
-        return cls._rt_possible_keys.copy()
+        return cls._rt_possible_keys
 
-    @classmethod
-    def get_possible_ri_keys(cls) -> List[str]:
-        """Method to get the supported retention index keys
+    def get_possible_ri_keys() -> List[str]:
+        """ A method that returns the possible keys to get retention indices.
 
         Returns:
-            List[str]: List of supported retention index keys.
+            List[str]:  A list of possible keys to get retention indices.
         """
-        return cls._ri_possible_keys.copy()
+        keys_conversions = load_known_key_conversions()
+        ri_key_converted = [ key for key, value in keys_conversions.items() if "retention_index" == value]
+        ri_key_converted.append("retention_index")
+        return ri_key_converted

```

Comment:
This should still be adapted to be in line with how the retention time keys are initialzied and obtained.

Reply:
Addressed [5ff2049](https://github.com/RECETOX/RIAssigner/pull/118/commits/5ff2049bc559c853cead7029770afef13430d25a)